code/game/objects/items/cosmetics.dm

```diff
@@ -107,6 +107,10 @@
 	flags_1 = CONDUCT_1
 	w_class = WEIGHT_CLASS_TINY
 
+/obj/item/razor/suicide_act(mob/living/carbon/user)

```

Comment:
doesn't leave you bald, terrible suicide

Reply:
Agreed

Reply:
shit u right